builtin/providers/aws/json.go

```diff
@@ -0,0 +1,38 @@
+package aws
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+func normalizeJson(jsonString interface{}) string {
+	if jsonString == nil || jsonString == "" {
+		return ""
+	}
+	var j interface{}
+	err := json.Unmarshal([]byte(jsonString.(string)), &j)
+	if err != nil {
+		return fmt.Sprintf("Error parsing JSON: %s", err)
+	}
+	b, err := json.Marshal(j)
+	if err != nil {
+		panic(err) // unexpected
+	}
+	return string(b)
+}
+
+func normalizePolicyDocument(policyString interface{}) string {
```

Comment:
Oh yeah... agreed that it's weird for `normalizeJson` to be doing it too, but we can certainly fix that separately.